.github/workflows/deploy-cpp.yml

```diff
@@ -175,9 +22,10 @@ jobs:
       name: "pypi"
       url: "https://pypi.org/project/awkward-cpp/"
     steps:
+
     - uses: actions/download-artifact@v4
       with:
-        pattern: "awkward*"
+        pattern: "awkward-cpp*"
```

Comment:
Again, using the `awkward-cpp` pattern so as to not pick up the `awkard` sdist and wheel.